### ISSUE TITLE
Issue with catch throw

### ISSUE THREAD

**Issue by bartgol** [cyrus-and/gdb-dashboard#296]
I am experiencing a crash when I try to catch exceptions thrown:
```
$ gdb ./field
...
Reading symbols from ./field...done.
>>> catch throw
Catchpoint 1 (throw)
>>> r
─── Output/messages ────
...
<blah blah output from program>
Thread 1 "field" hit Catchpoint 1 (exception thrown), 0x00007ffff5ba856e in __cxxabiv1::__cxa_throw (obj=0x10c4cd0, tinfo=0xf1acf8 <typeinfo for std::logic_error@@GLIBCXX_3.4>, dest=0x8ebdc0 <std::logic_error::~logic_error()@plt>) at /tmp/hrswant/spack-stage/spack-stage-gcc-10.1.0-x6hpom6kvatos2olzhipols76ettna47/spack-src/libstdc++-v3/libsupc++/eh_throw.cc:78
78	/tmp/hrswant/spack-stage/spack-stage-gcc-10.1.0-x6hpom6kvatos2olzhipols76ettna47/spack-src/libstdc++-v3/libsupc++/eh_throw.cc: No such file or directory.
Segmentation fault (core dumped)

$
```
Breakpoints work as expected, but catching exceptions seems to have problems. Notice that, without gdb-dashboard's .gdbinit, `catch throw` works as expected:
```
$ gdb ./field
...
Reading symbols from ./field...done.
(gdb) catch throw
Catchpoint 1 (throw)
(gdb) r
<blah blah output from program>
Thread 1 "field" hit Catchpoint 1 (exception thrown), 0x00007ffff5ba856e in __cxxabiv1::__cxa_throw (obj=0x10c4cd0, tinfo=0xf1acf8 <typeinfo for std::logic_error@@GLIBCXX_3.4>, dest=0x8ebdc0 <std::logic_error::~logic_error()@plt>)
    at /tmp/hrswant/spack-stage/spack-stage-gcc-10.1.0-x6hpom6kvatos2olzhipols76ettna47/spack-src/libstdc++-v3/libsupc++/eh_throw.cc:78
78	/tmp/hrswant/spack-stage/spack-stage-gcc-10.1.0-x6hpom6kvatos2olzhipols76ettna47/spack-src/libstdc++-v3/libsupc++/eh_throw.cc: No such file or directory.
(gdb) 
```

Is this something known/expected? Or is there a user error on my side? Thanks!

Some more info, in case it matters:
- OS: RHEL 8.7 (Ootpa)
- gcc: gcc (Spack GCC) 10.1.0
- gdb: GNU gdb (GDB) Red Hat Enterprise Linux 8.2-19.el8
- python: Python 3.9.13

**Comment by cyrus-and** [comment]
See #284 and linked issue.

TL;DR this is probably a GDB bug.

**Comment by bartgol** [comment]
Thanks for the quick reply. I will try to update my gdb to the latest version, as someone suggested on the linked issue.

Since it's not a problem with gdb-dashboard, we can close.

**Comment by cyrus-and** [comment]
Feel free to add details if you come up with something.
